src/samples/index.md

```diff
@@ -209,10 +208,10 @@ You might use the `Spacecraft` class like this:
 
 <?code-excerpt "misc/test/samples_test.dart (use class)"?>
 {% prettify dart %}
-var voyager = new Spacecraft('Voyager I', new DateTime(1977, 9, 5));
+var voyager = Spacecraft('Voyager I', DateTime(1977, 9, 5));
 voyager.describe();
 
-var voyager3 = new Spacecraft.unlaunched('Voyager III');
+var voyager3 = Spacecraft.unlaunched('Voyager III');
 voyager3.describe();
 {% endprettify %}
 
```

Comment:
In the [Read more] that follows, we should add "optional `new` and `const`" and delete "constant constructors".